spec/controllers/permissions_controller_spec.rb

```diff
@@ -54,6 +54,12 @@ def show
 
       expect(subject).to deny_access(redirect: sign_in_url)
     end
+
+    it 'denies access to show and display a flash message' do
```

Comment:
We're updating the style as we go forward. Please make the changes requested by hound.